src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,18 @@
-export const UserInfo = () => {};
+import { getUserById } from '../../services/todo';
+
+type Props = {
+  userId: number,
+};
+
+export const UserInfo: React.FC<Props> = ({ userId }) => {
+  const user = getUserById(userId);
+
+  return (
+    <a
+      className="UserInfo"
+      href={`mailto:${user?.email}`}
+    >
+      {user?.name}
+    </a>
+  );
+};
```

Comment:
You should create a modified array of todos with an additional `user` property rather than finding a user in this component.